janusgraph-server/src/test/resources/janusgraph-server-with-serializers.yaml

```diff
@@ -17,15 +17,14 @@ port: 8182
 scriptEvaluationTimeout: 30000
 channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
 graphs: {
-    graph: conf/janusgraph-inmemory.properties
+    graph: "src/test/resources/conf/janusgraph-inmemory.properties"
```

Comment:
Can you explain why this change is needed?

Reply:
I didn't like if test resources are in the directory `janusgraph-server/conf`.